docs/CONFIG-PROPERTIES.md

```diff
@@ -70,11 +66,37 @@
 | goroutine.leak.detection.keep.stats.hours | integer (hours) | 24 | Amount of hours to keep the stats for leak detection. We keep more stats than the check window to be able to react to settings with a bigger check window via configuration. |
 | goroutine.leak.detection.cooldown.minutes | integer (minutes) | 5 | Cooldown period in minutes after the leak detection is triggered. During this period, no stack traces are collected; only warning messages are logged. |
 
-In addition, there can be per-agent settings.
-The Per-agent settings begin with "agent.*agentname*.*setting*"
-The following per-agent settings override the corresponding default ones:
+
+## Log levels
+Log level can be set for three different components of EVE: zedbox, syslog, and kernel.
```

Comment:
zedbox is a process and a box wrapper around other items like tpmmgr etc which run as separate processes, so that isn't a good description. Furthermore, all of the linuxkit containers which send output to stdout (which presumably includes pkg/wlan, pkg/wwan, etc) appear in logread hence fall in that category of logging. So I think "EVE microservices" is a much better description than "zedbox".